index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Lembani Sakala's personal portfolio website.">
+    <title>Lembani Sakala - Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+        integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header class="header">
+        <nav class="nav">
+            <a class="brand" href="#">Welcome</a>
+            <a class="menu fa fa-bars fa-xl"></a>
+        </nav>
```

Comment:
[REQUIRED] Please fix the distance around the hamburger button 

![image](https://user-images.githubusercontent.com/74744787/158788488-7a4528af-47c4-4f27-8d5c-5d02aabf8cdc.png)
 
and please implement it so it match [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 
![image](https://user-images.githubusercontent.com/74744787/158789528-a5365b04-1168-4719-96c9-4f0a1588b200.png)

Reply:
In my commit, I have done what you suggest in the review ie
- **Fixed the spacing around the menu icon**
As shown below
<img width="766" alt="Screenshot 2022-03-17 at 1 28 08 PM" src="https://user-images.githubusercontent.com/30483991/158799614-aa10520d-c529-42d0-8f3e-e282e232b213.png">